x-pack/plugins/ml/public/application/datavisualizer/stats_datagrid/components/expanded_row_field_header/expanded_row_field_header.tsx

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiText } from '@elastic/eui';
+import React from 'react';
+
+export const ExpandedRowFieldHeader = ({ children }: { children: React.ReactNode }) => (
+  <EuiText size="xs" color={'subdued'} className={'mlFieldDataCard__valuesTitle'}>
+    {children}
+  </EuiText>
```

Comment:
I think we can remove this class altogether with some slight tweaks to the design.  Discover does not color these type of headings, or center them. So I think we can simply do:

```
<EuiTitle size="xxxs">
    <h4>{children}</h4>
</EuiTitle>
```

Reply:
I think we will keep it like this for now with the class to remain consistent with the file-based data visualizer.